### DISCUSSION TITLE
Supabase giving "Unsupported Provider" even after enabling provider in the settings

### DISCUSSION THREAD

**Discussion by thegauravthakur** [supabase/supabase#7402] [Questions] [answered]
Hello team, I'm new to Supabase. 

The "Google" provider has been enabled in the Supabase authentication settings and I have entered the correct credentials, but it is still giving the error "Unsupported provider: Provider is not enabled" when I navigate to "https://tilsueiaeyqyjwebdbbp.supabase.co/auth/v1/authorize?provider=google". 

Could anyone please let me know what I'm missing?

**Update**:
It was a server location issue; earlier it was in India, but once I updated it to USA, it worked

**Comment by thegauravthakur** [comment, marked as answer]
It was a server location issue; earlier it was in India, but once I updated it to USA, it worked

**Reply by luisbazdev** [reply]
Can you give more details on how you fixed it? I have the same problem

**Reply by GaryAustin1** [reply]
There was an update in a Discord issue from an SB employee this morning:
`It looks like the unsupported provider error may be an issue on our side.  Our Auth team is looking at it today.`
https://discord.com/channels/839993398554656828/990268349592248360
I've not seen an update yet.

**Comment by pcardosolei** [comment]
I have issues on this. 

In Facebook and Google.

The workaround is to change the server place?

**Reply by luisbazdev** [reply]
I changed the server location and it worked for me.

**Comment by valekar** [comment]
Is there any update on this issue ? I am facing the same issue.

**Reply by valekar** [reply]
We have already hosted our database by paying price to it. Now is there is a way to change the location of it ? We have hosted it in central europe
